src/components/Page/Secrets/Secrets.test.tsx

```diff
@@ -29,34 +30,14 @@ describe('Secrets component', () => {
 	test('renders Secrets component with title and location', () => {
 		const { getByText } = render(<Secrets data={mockData} />)
 
-		expect(getByText('Secrets Vulnerability')).toBeInTheDocument()
-		expect(getByTextAcrossMultipleElements(getByText, 'Location: example-file.yml: 10'))
+		expect(getByText(mockData.header)).toBeInTheDocument()
+		expect(getByTextAcrossMultipleElements(getByText, 'Location: example-file.yml'))
 	})
 
 	test('renders Secrets component with severity and abbreviation', () => {
-		const { getByText } = render(<Secrets data={mockData} />)
-
-		expect(getByTextAcrossMultipleElements(getByText, 'Severity: Medium'))
-		expect(getByTextAcrossMultipleElements(getByText, 'Abbreviation: XYZ'))
-	})
-
-	test('renders Secrets component with code finding', () => {
-		const { getByText } = render(<Secrets data={mockData} />)
-
-		expect(getByText('Example code snippet')).toBeInTheDocument()
-		expect(getByText('FINDING')).toBeInTheDocument()
-		expect(getByText('How it happens')).toBeInTheDocument()
-		expect(getByText('Meaning of the vulnerability')).toBeInTheDocument()
-		expect(getByText('What to do about it')).toBeInTheDocument()
-	})
-
-	test('renders Secrets component with description', () => {
-		const { getByText } = render(<Secrets data={mockData} />)
-
-		expect(getByText('What should I do?')).toBeInTheDocument()
-		expect(getByText('What can happen?')).toBeInTheDocument()
-		expect(getByText('What Does it mean?')).toBeInTheDocument()
-		expect(getByText('Description')).toBeInTheDocument()
-		expect(getByText('This is a vulnerability description.')).toBeInTheDocument()
```

Comment:
Let's add new tests in place of those, which cover the visibility of InformationTabs and Header